### PR TITLE
tools/run-container: drop support for python2

### PR DESCRIPTION
**Pull request by paride**
Drop support for specifying an Python interpreter different from python3
from tools/run-container.

### CODE REVIEW

**Comment by paride** [comment]
Tested with:
```
/tools/run-container centos/7 --source-package --artifacts=.
```

**Comment by igalic** [comment]
what do i do when the python 3 on my distro isn't called `python3`?

oh, never mind that's `run-container`, we can't run those on FreeBSD anyway :sweat_smile:

File: tools/run-container [hunk 1]
```diff
@@ -367,7 +347,7 @@ wait_for_boot() {
     run_self_inside "$name" wait_inside "$name" "$wtime" "$VERBOSITY" ||
         { errorrc "wait inside $name failed."; return; }
 
-    if [ ! -z "${http_proxy-}" ]; then
+    if [ -n "${http_proxy-}" ]; then
```
Comment (smoser) [inline tools/run-container]: this is unrelated.

Comment (paride) [reply tools/run-container]: commit dropped.
